lib/roger_jshint/generator.rb

```diff
@@ -0,0 +1,44 @@
+require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.